snap/snapcraft.yaml

```diff
@@ -85,3 +85,8 @@ apps:
       - screen-inhibit-control
       - unity7
       - camera
+
+plugs:
+  browser-sandbox:
+    interface: browser-support
+    allow-sandbox: true
```

Comment:
Can you add a newline at the end of the file? This is required for text files on Unix. It makes it easier to look at files using tools like `cat` and some apps bork when a file ends without a newline.

Reply:
Done! Thanks :)